.github/workflows/quay-tag.yml

```diff
@@ -0,0 +1,38 @@
+name: quay.tag
+run-name: Creating new tag in quay based on pushed tag in Github.
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  copy-tag-to-quay:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: '0'
+      - name: Install skopeo
+        shell: bash
+        run: |
+          sudo apt-get -y update
+          sudo apt-get -y install skopeo
+      - name: Login to quay.io
+        shell: bash
+        env:
+          QUAY_TOKEN: ${{ secrets.QUAY_ROBOT_TOKEN }}
+        run: |
+         skopeo login quay.io -u opendatahub+odh_dashboard -p ${QUAY_TOKEN}
+      - name: Get latest tag name
+        id: tag
+        run: echo "tag=$(git describe --tags)" >> $GITHUB_OUTPUT
+      - name: Get latest hash
+        id: hash
+        run: echo "hash=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
+      - name: Create new tag
+        shell: bash
+        env:
+          TAG: ${{ steps.tag.outputs.tag }}
+          HASH: ${{ steps.hash.outputs.hash }}
+        run: |
+          skopeo copy docker://quay.io/opedatahub/odh-dashboard:main-${{ env.HASH }} docker://quay.io/opendatahub/odh-dashboard:${{ env.TAG }}
```

Comment:
Typo in the name and it would be good to move the repo url to a variable to make the code portable